noether/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ find_package(catkin REQUIRED cmake_modules
     vtk_viewer
     noether_msgs
     roscpp
+    pcl_conversions
```

Comment:
Is this required if you are using pcl/pcl_conversion.h?

Reply:
The header in the pcl library is "pcl/conversions.h" and it has different functions than those found in the ROS "pcl_conversion.h" header file